test/e2e/multicast_test.go

```diff
@@ -40,8 +42,11 @@ func skipIfMulticastDisabled(tb testing.TB) {
 }
 
 var igmpQueryType = int32(0x11)
+var externalHostIdx = 0
+var externalHostIface string
 
 func TestMulticast(t *testing.T) {
+	skipIfNoExternalHosts(t)
```

Comment:
I think we can have a single case dedicated for traffic with external host, and skip the single case if no external hosts is existing in the cluster, rather than skip all multicast test cases